README.md

```diff
@@ -118,6 +121,9 @@ $result = $q->execute(array(
 ```
 
 ### Using PostgreSQL 9.3+ JSON operators
+
+Note that you need to use the function names. This library does not add support for custom operators like `@>`.
```

Comment:
maybe this should be right at the top - don't know how it is for mysql. or if i missed something?

Reply:
I expect the users of this library to understand that writing DQL is different from SQL. 
There are more DQL operators that deviate from SQL or not even exist in SQL at all like `IDENTITY()`.
I don't think a specific dbms note should be that important.

Reply:
i had hoped that one can also add custom operators to DQL, as DQL does support some comparison operators in DQL. so i think it has value to mention that that does not work. but it belongs into a general section as its not about postges, but about extending DQL in general.

but if you want me to drop the note completely, i can also do. i personally will now probably remember :-)

Reply:
I think the note here is fine. I just commented on the "right at the top" part.